jobspec2/types.config.go

```diff
@@ -315,18 +314,32 @@ func (c *jobConfig) EvalContext() *hcl.EvalContext {
 			end := t.SourceRange().End.Byte
 
 			v := string(body[start:end])
-			if i := bytes.IndexByte(body[end:], '}'); i != -1 {
-				v += string(body[end : end+i+1])
-			} else {
-				// fallback for unexpected cases
-				v += "}"
+
+			// find the start of inclusing "${..}" if it's inclused in one; otherwise, wrap it in one
+			quoted := false
```

Comment:
I'm finding this a little hard to follow and maybe it's the name of this variable that's confusing me. Is this really only signifying "quoted" or is it doing double-duty for something like `isBracketed` as well?

Reply:
Bracketed is a better name - didn't know what to call it!  Thanks!